### PR TITLE
Releasing 2.8.0rc7

### PR DESCRIPTION
**Pull request by unho**
Releasing 2.8.0rc7

### CODE REVIEW

File: docs/server/commands.rst [hunk 1]
```diff
@@ -114,6 +114,29 @@ queue you can run this command.
...
+specific languages or projects.
+
+.. django-admin-option:: --store
+
+Use the :option:`--store` option to narrow the stats data calculation to a
+specific store:
+
+.. code-block:: console
+
+    $ pootle update_data --store=fr/proj/mydir/mystore.po
```
Comment (phlax) [inline docs/server/commands.rst]: i think this needs a preceeding slash - not sure

Comment (unho) [inline docs/server/commands.rst]: Yes, will change it.

**Comment by phlax** [comment]
lgtm, once path issue is resolved

File: docs/releases/2.8.0.rst [hunk 2]
```diff
@@ -288,6 +298,9 @@ Command changes and additions
   :option:`--rotate <export --rotate>` option allows old files to be removed.
 - Added :djadmin:`schema` command that allows to dump the database schema on
   MySQL which is useful for diagnosing differences in database schema.
+- Added :djadmin:`update_data` command that allows to update the stats data.
+- :djadmin:`init` now creates a development configuration with
+  :option:`--rotate <export --rotate>` option.
```
Comment (phlax) [inline docs/releases/2.8.0.rst]: i think this rotate option is duplicated

**Pull request merged**